draft-ietf-trans-rfc6962-bis.md

```diff
@@ -552,14 +552,15 @@ Tree no later than a fixed amount of time, known as the Maximum Merge Delay
 (MMD), after the issuance of the SCT. Periodically, the log MUST append all its
 new entries to its Merkle Tree and sign the root of the tree.
 
-Log operators MUST NOT impose any conditions on retrieving or sharing data from
-the log.
+Log operators SHOULD NOT impose any conditions on retrieving or sharing data
+from the log.
 
 ## Accepting Submissions
 
-Before accepting a submitted certificate or precertificate, the log MUST verify
-that it has a valid signature chain to an accepted trust anchor, using the chain
-of intermediate CA certificates provided by the submitter. Logs SHOULD accept
+To avoid logs being overloaded by invalid submissions, before accepting a
+submitted certificate or precertificate, the log MUST verify that it has a
+valid signature chain to an accepted trust anchor, using the chain of
+intermediate CA certificates provided by the submitter. Logs SHOULD accept
```

Comment:
LGTM - very precise about forbidding the log from accepting a submission even if it can build alternative chains, which I like.